cli/internal/taskhash/taskhash.go

```diff
@@ -174,6 +174,8 @@ func (th *Tracker) CalculateFileHashes(allTasks []dag.Vertex, workerCount int, r
 			continue
 		}
 
+		// TODO(mehulkar): Once we start composing turbo.json, we need to change this
+		// to look in the graph for TaskDefinitions, rather than the root pipeline.
```

Comment:
I have another ticket for this. It shouldn't break anything since there is only one turbo.json so far.